docs/releasing/header.txt

```diff
@@ -1,19 +1,13 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
**Thought:**
I guess this means that the release manifests will always use the current year?
That is, if we introduce a new file inside `kubernetes-manifest` in 2024, we would have to bump `2023` here to `2024`.
Fine by me.

Reply:
My take here is that technically, every file in `kubernetes-manifests/` is _always_ a new file, because it's newly generated / rendered from scratch at release time, as opposed to other more static files (like from `src/`) which wouldn't have their header changing and would keep the "earlier" date.